tests/indices.spec.ts

```diff
@@ -0,0 +1,124 @@
+import { client } from './dbConnection';
+import { expect } from '@jest/globals';
+import FalkorDB from '../src/falkordb';
+import Graph from '../src/graph';
+
+describe('Indices Tests', () => {
+    let clientInstance: FalkorDB;
+    let graphName: Graph;
+
+    beforeAll(async () => {
+        try {
+            clientInstance = await client();
+        } catch (error) {
+            console.error('Failed to initialize database connection:', error);
+            throw error;
+        } 
+    });
+
+    afterAll(async () => {
+        try {
+            await clientInstance.close()
+        } catch (error){
+            console.error('Failed to close database connection:', error);
+            throw error;
+        }
+    });
+
+    beforeEach(async () => {
+        try {
+            graphName = clientInstance.selectGraph("graph");
+        } catch (error) {
+            console.error('Failed to select graph:', error);
+            throw error;
+        }
+    })
+
+    afterEach(async () => {
+        try {
+            await graphName.delete()
+        } catch (error) {
+            console.error('Failed to delete graph:', error);
+            throw error;
+        }
+    })
+
+    it('Create and verify indices on a node label', async () => {
+        await graphName.query(`CREATE (:label1)`);
```

Comment:
can we have api to create/drop index like in https://github.com/FalkorDB/falkordb-py/blob/d0671f61c5a8341a11635d2f705462c07ba2b4a7/falkordb/graph.py#L457

Reply:
Done